src/server/server.ts

```diff
@@ -128,6 +147,47 @@ async function serveArticle(req: Request, res: ExpressResponse): Promise<void> {
     }
 }
 
+const liveBlockUpdates = (since: number, content: Content): LiveUpdates => ({
+    newBlocks: serialiseLiveBlocks(newBlocksSince(since)(content)(docParser)),
+    updatedBlocks: serialiseLiveBlocks(updatedBlocksSince(since)(content)(docParser)),
+});
+
+const recentLiveBlocks = (content: Content): LiveUpdates => ({
+    newBlocks: serialiseLiveBlocks(recentBlocks(10)(content)(docParser)),
+    updatedBlocks: [],
+});
+
+async function liveBlocks(req: Request, res: ExpressResponse): Promise<void> {
+    try {
+        const articleId = req.params.articleId || defaultId;
+        const since = parseNumber(req.query.since);
```

Comment:
I think this should be a date. It's fine while developing, but in prod we should be consistent with MAPI

Reply:
Of the format: `2020-05-21T10:00:00Z`?

Reply:
yep!

Reply:
Done!